dev/include_dune

```diff
@@ -0,0 +1,22 @@
+(* File to include to install the pretty-printers in the ocaml toplevel *)
+
+(* Typical usage :
+
+   $ dune exec coqtop.byte # or even better : rlwrap coqtop.byte
+   Coq < Drop.
+   # #directory "dev";;
+   # #use "include";;
+
+   Alternatively, you can avoid typing #use "include" after each Drop
+   by adding the following lines in your $HOME/.ocamlinit :
```

Comment:
Since we already ship a `.ocamlinit` file, wouldn't it make sense to include these lines there?

Reply:
IMHO not as the included file may have other uses; for example utop will install printers differently [and automatically once we annotate them].